doc/src/min_modify.txt

```diff
@@ -13,11 +13,13 @@ min_modify command :h3
 min_modify keyword values ... :pre
 
 one or more keyword/value pairs may be listed :ulb,l
-keyword = {dmax} or {line} or {alpha_damp} or {discrete_factor}
+keyword = {dmax} or {line} or {norm} or {alpha_damp} or {discrete_factor}
   {dmax} value = max
     max = maximum distance for line search to move (distance units)
-  {line} value = {backtrack} or {quadratic} or {forcezero}
-    backtrack,quadratic,forcezero = style of linesearch to use 
+  {line} value = {backtrack} or {quadratic} or {forcezero} or {spin_cubic} or {spin_none}
+    backtrack,quadratic,forcezero,spin_cubic,spin_none = style of linesearch to use 
+  {norm} value = {euclidean} or {max}
```

Comment:
I suggest:
norm value = two or max
  two = Euclidean two-norm (length of 3N vector)
  max = max absolute value of any 3N vector component

I think "max" is the option you are adding?  As Aidan asked (I think)
is this the true max norm (across all components), or
are you taking the length of the 3-vector for each atom (2-norm)
and taking the max of those across all atoms?  That should be explained below.

Reply:
And the name of what I called the true max norm is the infinity norm, correct?
If that's what you want, best to call it that.